MediaManager.UWP/AudioPlayerImplementation.cs

```diff
@@ -165,9 +165,14 @@ public Task Pause()
             try
             {
                 var sameMediaFile = mediaFile == null || mediaFile.Equals(_currentMediaFile);
-                if (Status == MediaPlayerStatus.Paused && sameMediaFile)
+                var currentMediaPosition = _player.PlaybackSession?.Position;
+                // This variable will determine whether you will resume your playback or not
+                var resumeMediaFile = Status == MediaPlayerStatus.Paused && sameMediaFile ||
+                                      currentMediaPosition?.TotalSeconds > 0 && sameMediaFile;
+                if(resumeMediaFile)
                 {
-                    _player.PlaybackSession.PlaybackRate = 1;
+                    // TODO: PlaybackRate needs to be configurable rather than hard-coded here
```

Comment:
Hi @martijn00 , I think it's some works required to get this done as it needs to be defined in abstraction layer and implements per platform (iOS, Android & UWP). I will revert it to hard-coded one for now if you don't mind. but will create a new pull request to control playback speed later on